### PR TITLE
systemctl list-dependencies: fix duplicate output and take multiple arguments

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM.

File: src/systemctl/systemctl.c [hunk 1]
```diff
@@ -1786,30 +1788,40 @@ static int list_dependencies_one(
 }
 
 static int list_dependencies(int argc, char *argv[], void *userdata) {
-        _cleanup_strv_free_ char **units = NULL;
+        _cleanup_strv_free_ char **units = NULL, **done = NULL;
         _cleanup_free_ char *unit = NULL;
```
Comment (yuwata) [inline src/systemctl/systemctl.c]: `unit` is not used. Please drop this.

Comment (keszybz) [reply src/systemctl/systemctl.c]: Thanks, updated.

**Comment by keszybz** [comment]
Taking the liberty to add back the green flag. The only change is the removal of the unused variable.

**Pull request merged**